eth/sync.go

```diff
@@ -271,50 +273,50 @@ func (cs *chainSyncer) nextSyncOp() *chainSyncOp {
 	}
 	if mode == downloader.FastSync {
```

Comment:
I think there's a mistake in the merge here. This check for FastSync and the associated code block appears to have now been removed from upstream, so should it also be removed from here?